core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -288,9 +293,18 @@ protected void startServingRPCServer() {
     try {
       transportFactory = mTransportProvider.getServerTransportFactory();
     } catch (IOException e) {
-      throw Throwables.propagate(e);
+      throw new RuntimeException(e);
     }
 
+    try {
+      if (mTServerSocket != null) {
```

Comment:
This is not null when the AlluxioMasterProcess is created since mTServerSocket is also created in the ctor (to get the socket address in case of 0-port).